src/main/riddl/Test/messages.riddl

```diff
@@ -5,7 +5,7 @@ command CreateTenants is {
 | provided. The Info has only basic contact information.
 }
 
-event TenantsCreated is {
+result TenantsCreatedResult is {
```

Comment:
This confuses me. TenantsCreated is an event that is generated in response to a command, right? Results are the response to a Query.